clr-installer/main.go

```diff
@@ -325,6 +325,58 @@ func execute(options args.Args) error {
 		return nil
```

Comment:
This `return nil` in the original code would return from the `execute` function and stop processing; this is a logic error that needs to be fixed.

Reply:
Make sense .. Will do